clouddriver-aws/src/test/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/handlers/BasicAmazonDeployHandlerUnitSpec.groovy

```diff
@@ -291,6 +293,37 @@ class BasicAmazonDeployHandlerUnitSpec extends Specification {
     actualClassicLinkVpcId == null
   }
 
+  void "should not modify unlimited cpu credits if not applicable"() {
+    setup:
+    def unlimitedCreditsInput = null
```

Comment:
probably worth sticking in a `where:` block and testing a few permutations of this value

Reply:
I didn't add the other values here because those invalid request cases where unlimitedCpuCredits is not null and bursting isn't supported will fail validation [here](https://github.com/spinnaker/clouddriver/blob/4201dff3032643867a7ce2a92c6b5b0e1eda41e7/clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/validators/BasicAmazonDeployDescriptionValidator.groovy#L83).